client/src/Components/CardContainer/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from 'antd';
+import Card from '../Common/Card';
+import './style.css';
+
+const { Title, Text } = Typography;
+const CardContainer = ({ title, data }) => (
+  <div>
+    <Title level={3} id="container-title">
+      {title}
+    </Title>
+    <div className="container-cards">
+      <ul>
+        {data.length !== 0 ? (
+          data.map((item) => (
+            <li>
```

Comment:
```suggestion
            <li key={id} >
```

Reply:
Here I think you should take the kindergartenId from the props (data) and put it as a key for the li